src/modules/admin/components/formRules/FormRuleCondition.tsx

```diff
@@ -0,0 +1,88 @@
+import { Grid, Typography } from '@mui/material';
+import { SxProps } from '@mui/system';
+import { startCase } from 'lodash-es';
+import React, { ReactNode } from 'react';
+import { useIsMobile } from '@/hooks/useIsMobile';
+import { ConditionType } from '@/modules/admin/components/formRules/NewFormRuleDialog';
+import FormSelect from '@/modules/form/components/FormSelect';
+import { isPickListOption } from '@/modules/form/types';
+import { PickListOption } from '@/types/gqlTypes';
+
+const FormRuleLabelTypography = ({
+  sx,
+  children,
+}: {
+  sx?: SxProps;
+  children: ReactNode;
+}) => (
+  <Typography variant='body2' sx={{ pt: 4.25, ...sx }}>
```

Comment:
Looks like we lost the `fontWeight: 600` here.

The 4.25 thing is fine. Just a learning-oriented comment: another way it could be done is `py: 1` with  `alignItems='flex-end'` on the Grid container. I think that feels slightly less janky to me but they look pretty much identical.

Reply:
Oh, thanks for this, I agree that is much less janky, ha!

Regarding `fontWeight`, yes, I removed this on purpose to visually distinguish the "logical flow" label text from the form element label text, but it was a pretty arbitrary decision. I'll loop in @bobbyw-ng for a preference between these 2 screenshots:

1. With all labels bold

<img width="872" alt="Screenshot 2024-08-09 at 12 01 09 PM" src="https://github.com/user-attachments/assets/8aca6f8a-8500-49da-b112-15937f339dea">


3. With only Form Control labels bold, and "logical flow" labels in normal weight

<img width="862" alt="Screenshot 2024-08-09 at 12 00 53 PM" src="https://github.com/user-attachments/assets/dc0f76b0-f95a-4da9-a0b0-1733c12b0d51">

Reply:
Oh alright no preference, just pointing it out in case it was accidentally missed. I think it looks fine as you have it without the bold!

Reply:
I agree I think it's fine without the bold; it would be more consistent with the rest of the site as we typically use bold for labels.

This is very minor but I wonder if these should be all caps, since that's typically how logical operators are formatted (IF instead of If, AND instead of And).

Reply:
I have a preference for keeping the normal case since it matches the "display sentence" of the form rule, shown at the top. I guess we could change it there too, like this:
<img width="849" alt="Screenshot 2024-08-13 at 11 20 34 AM" src="https://github.com/user-attachments/assets/3155b6f9-bfd5-4357-b11d-299f1f77c2a0">

But I think there are downsides to doing that, because there is some non-logical-operator text included there too.